test/test-match-conditions.js

```diff
@@ -34,8 +36,24 @@ function revertShim() {
   isMatch.__set__('version', versionCache);
   isMatch.__set__('platform', platformCache);
   isMatch.__set__('semVersion', semVersionCache);
+  isMatch.__set__('distro', distroCache);
+  isMatch.__set__('release', releaseCache);
 }
 
+test('testCache', function (t) {
+  t.ok((typeof platformCache !== 'undefined' && platformCache),
```

Comment:
this test seems odd. if platformCache were undefined it would fail the test, I don't think we need to test both of these